builder/proxmox/common/config.go

```diff
@@ -97,6 +97,10 @@ type Config struct {
 	// `debian-12;template`.
 	Tags string `mapstructure:"tags"`
 
+	// Arbitrary arguments passed to KVM. For example
+	// ` -no-reboot -smbios type=0,vendor=FOO`.
+	// 	Note: this option is for experts only.
+	Args string `mapstructure:"args"`
```

Comment:
Bit nitpicky on my part, but just for clarity, since this is for extra raw args, and since this is for advanced use-cases only, I would suggest making the name more explicit here, something along the lines of `qemu_extra_args`.

Reply:
Sure. Renamed it to `qemu_additional_args` (to stay consistent with e.g. `additional_iso_files`).
Renamed the internal structure-property to `AdditionalArgs`.

Reply:
FYI: Succesfully tested it manually with the new name.

Reply:
Sure. Renamed it to `qemu_additional_args` (to stay consistent with e.g. `additional_iso_files`).
Renamed the internal structure-property to `AdditionalArgs`.

Reply:
FYI: Succesfully tested it manually with the new name.